### ISSUE TITLE
Using git duet for reverts?

### ISSUE THREAD

**Issue by kindrowboat** [git-duet/git-duet#16]
Is it possible to use git duet when reverting?  Right now when I have pair set up, and run `git revert <some_sha>` I see the following

```
Committer: [REDACTED USER] <[REDACTED USER]@[REDACTED HOSTNAME].[REDACTED DOMAIN]>
Your name and email address were configured automatically based
on your username and hostname. Please check that they are accurate.
You can suppress this message by setting them explicitly:

    git config --global user.name "Your Name"
    git config --global user.email you@example.com

After doing this, you may fix the identity used for this commit with:

    git commit --amend --reset-author
```

**Comment by jszwedko** [comment]
@aauthor currently there is not, I think this is a good case for `git duet-revert` though. I'll see about implementing that soon.

**Comment by jszwedko** [comment]
Released as 0.3.0, thanks for raising the issue @aauthor !
